extensions/panache/hibernate-orm-panache-kotlin/runtime/pom.xml

```diff
@@ -71,6 +71,22 @@
         </dependency>
     </dependencies>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>jcenter</id>
```

Comment:
Is `dokka` only on JCenter and not on Maven Central?

Reply:
correct